src/lib/realtime.js

```diff
@@ -134,27 +136,42 @@ function getCozySocket(cozy) {
       }
     }
 
-    const onSocketClose = async event => {
+    const onSocketClose = async (event, numRetries) => {
       if (!event.wasClean) {
         console.warn &&
           console.warn(
             `WebSocket closed unexpectedly with code ${event.code} and ${event.reason
               ? `reason: '${event.reason}'`
-              : 'no reason'}. Reconnecting.`
+              : 'no reason'}.`
           )
-        try {
-          socket = await connectWebSocket(cozy, onSocketMessage, onSocketClose)
-        } catch (error) {
-          console.error &&
-            console.error(
-              `Unable to reconnect to realtime. Error: ${error.message}`
+
+        if (numRetries) {
```

Comment:
Could you add a little timer before reconnecting on a retry ?

Reply:
@jinroh Great idea. Which amount of time seems good to you ? 500ms ? 1s ?

Reply:
1 second should be good.

`NUM_RETRIES - numRetries + 1` seconds could be better, with more retries.

Reply:
Nice suggestion, love it. 👍